DataLakeModels/Models/Reels/Reel.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace DataLakeModels.Models.Reels {
+    public class Reel : IEquatable<Reel> {
+        public string Id { get; set; }
+        public string Pk { get; set; }
+        public string Code { get; set; }
+        public User User { get; set; }
+        public string UserId { get; set; }
+        public Caption Caption { get; set; }
```

Comment:
Maybe we can keep all the entity relationships (foreign keys) at the top, for better readability